Gemfile

```diff
@@ -18,6 +18,7 @@ gem 'hiredis'
 gem 'httparty'
 gem 'lograge'
 gem 'newrelic_rpm'
+gem 'nokogiri' # required for premailer-rails
```

Comment:
The reason why the premailer gem says to add `nokogiri` or `hpricot` to the Gemfile is because it doesn't already include them as dependencies. It leaves it up to the user to add them. Since we already have `nokogiri` via other gems that do depend on it, we don't need to explicitly specify it in the Gemfile. My guess is that the issue you are seeing is due to something else.